src/collections/members/neil-alvares/index.mdx

```diff
@@ -0,0 +1,10 @@
+---
+layout: person
```

Comment:
@Anita-ihuman This is the previous template. Follow the template for the `layer5-ng` site.

Reply:
template here - https://github.com/layer5io/layer5/blob/layer5-ng/src/collections/members/_member-profile-template/index.mdx